cluster/upgrade_tool.sh

```diff
@@ -0,0 +1,19 @@
+tool=$1
+version=$2
+full=${tool}-${version}
+local_jar="${full}.jar"
+if [ -n "$3" ]; then
+  local_jar="$3"
+else
+  wget "https://github.com/hartwigmedical/hmftools/releases/download/${tool}-v${version}/${full}.jar"
+fi
+gsutil cp $local_jar "gs://common-tools/${tool}/${version}/${tool}.jar"
+if [ -z "$3" ]; then
+  rm local_jar
+fi
+
+./images/create_custom_image.sh -t
```

Comment:
Would prefer something a little more robust here, like `"$(dirname $0)/images/create_custom_image.sh -t"` so we've made a reasonable effort to allow it to be run from outside the directory it lives in.

Reply:
it needs to run in the directory of the poms